src/blackmisc/simulation/fscommon/fsdirectories.h

```diff
@@ -110,18 +126,37 @@ namespace BlackMisc::Simulation::FsCommon
         //! Get all the SimObjects paths from all config files
         static QSet<QString> allFsxSimObjectPaths();
 
+        // TODO TZ
+        //! Get all the SimObjects paths from all config files
+        static QSet<QString> allMsfsSimObjectPaths();
+
         //! Find the config files (fsx.cfg)
         // C:/Users/Joe Doe/AppData/Roaming/Microsoft/FSX/fsx.cfg
         static QStringList findFsxConfigFiles();
 
+        // TODO TZ
+        //! Find the config files (fsx.cfg)
+        // C:/Users/Joe Doe/AppData/Roaming/Microsoft/FSX/fsx.cfg
+        static QStringList findMsfsConfigFiles();
+
         //! Get all the SimObjects paths from fsx.cfg
         // SimObjectPaths.0=SimObjects\Airplanes
         static QSet<QString> fsxSimObjectsPaths(const QStringList &fsxFiles, bool checked);
 
+        // TODO TZ
+        //! Get all the SimObjects paths from msfs.cfg
+        // SimObjectPaths.0=SimObjects\Airplanes
+        static QSet<QString> msfsSimObjectsPaths(const QStringList &msfsFiles, bool checked);
+
         //! Get all the SimObjects files from fsx.cfg
         // SimObjectPaths.0=SimObjects\Airplanes
         static QSet<QString> fsxSimObjectsPaths(const QString &fsxFile, bool checked);
 
+        // TODO TZ
```

Comment:
Is this TODO still outstanding?

Reply:
I think this function doesn't do what it's supposed to do. That's why I want to watch it